### ISSUE TITLE
Documentation on calling methods

### ISSUE THREAD

**Issue by jingzans** [handsontable/ngHandsontable#34]
Hi,

Could you write up some documentation on how we can call specific methods? 

e.g. getDataAtRow, loadData, alter , etc.

Thanks,
Johnny

**Comment by hash1492** [comment]
If anyone has found the solution please do reply! How to call those methods for angular handsontable?

**Comment by HandsTest** [comment]
Not sure if its the right way to do. But we used afterInit method to get HOT instance and called the specific methods. this trick seemed to work for us.

$scope.afterInit = function() {
            // get the handsontable instance to use
            $scope.handsTableInstance = this;
        };

if ($scope.handsTableInstance) {
                    $scope.handsTableInstance.loadData($scope.items);
        }

**Comment by ares86** [comment]
Yeah, I have had some issue understanding how to implement a few thing. It would be helpful if some documentation is provided.

**Comment by uggel** [comment]
HandsTest, yes that works.
